#X button#
bool x_pressed false
#Y button#
bool y_pressed false
#A button#
bool a_pressed false
#B button#
bool b_pressed false
#left trigger#
bool lt_pressed false
#right trigger#
bool rt_pressed false
#LB trigger#
bool lb_pressed false
#RB trigger#
bool rb_pressed false
#d-pad down#
bool dd_pressed false
#d-pad up#
bool du_pressed false
#d-pad left#
bool dl_pressed false
#d-pad right#
bool dr_pressed false
#left stick pressed#
bool l3_pressed false
#right stick pressed#
bool r3_pressed false
#back button#
bool back_pressed false
#start button#
bool start_pressed false
#right stick#
StickPosition left_stick
#left stick#
StickPosition right stick